src/main/java/sirius/kernel/health/HandledException.java

```diff
@@ -19,24 +25,31 @@
  */
 public class HandledException extends RuntimeException {
 
+    @Serial
     private static final long serialVersionUID = 4460968279048068701L;
 
+    private final transient Map<ExceptionHint, Object> hints;
+
     /**
      * Creates a new instance with the given message and no exception attached
      *
      * @param message the message to be shown to the user
+     * @param hints   any processing hints which have been specified so far
+     * @param ex      the exception to be attached
      */
-    protected HandledException(String message) {
-        super(message);
+    protected HandledException(String message, Map<ExceptionHint, Object> hints, Throwable ex) {
```

Comment:
Optional: `ex` in `exception` oder `throwable` umbenennen. (Github lässt mich leider nicht suggesten an der Stelle :( )